site/en/workflows.md

```diff
@@ -1220,16 +1404,46 @@ copies them the a new Output fileGrp, re-generating the PAGE XML from the curren
 # Recommendations
 
 <!-- BEGIN-INCLUDE ./repo/ocrd-website.wiki/Workflow-Guide-recommendations.md -->
-All processors, with the exception of those for post-correction, were tested on
-selected pages of some prints from the 17th and 18th century.
+In order to facilitate the usage of OCR-D and the configuration of workflows, we provide two workflows
+which can be used as a start for your OCR-D-tests. They were determined by testing the processors listed
+above on selected pages of some prints from the 17th and 18th century.
 
 The results vary quite a lot from page to page. In most cases, segmentation is a problem.
 
-These recommendations may also work well for other prints of those centuries.
-
 Note that for our test pages, not all steps described above werde needed to obtain the best results.
 Depending on your particular images, you might want to include those processors again for better results.
 
+We are currently working on regression tests with the help of which we will be able to provide more profound
+workflows soon, which will replace those interim solutions. 
+
+## Minimal workflow
+
+Since `ocrd-tesserocr-recognize` can do binarization (Otsu), region
+segmentation, table recognition, line segmentation and text recognition at once, just like the
+upstream `tesseract` command line tool, it's a good single-step workflow to get
+a baseline result to compare to granular workflows.
+
+**Note:** Be aware that you will most likely obtain significantly better
+results by configuring a more granular workflow like e.g. the
+[workflows](#best-results-for-selected-pages)
+[below](#good-results-for-slower-processors).
+
+<table class="processor-table">
+  <thead>
+    <tr>
+      <th>Step</th>
+      <th>Processor</th>
+      <th>Parameter</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>1</td>
+      <td>ocrd-tesserocr-recognize</td>
+      <td>-P segmentation_level region -P textequiv_level word -P find_tables true -P model GT4HistOCR_50000000.997_191951</td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
is there a call I can simply copy and paste somewhere?

Reply:
You mean an _Example with ocrd-process_? Just copy the content of the table:

```suggestion
</table>

### Example with ocrd-process

    ocrd process "tesserocr-recognize -P segmentation_level region -P textequiv_level word -P find_tables true -P model GT4HistOCR_50000000.997_191951"

```

(Using space-indented command code syntax here, because code block inside suggestion block doesn't work; fingers crossed.)

Reply:
perfect :+1:

Reply:
> perfect

Did you want to commit the suggestion?